UM/Qt/qml/UM/MessageStack.qml

```diff
@@ -91,16 +92,122 @@ ListView
                 margins: UM.Theme.getSize("default_margin").width
             }
 
-            height: childrenRect.height
+            height: Math.max(messageTypeIcon.height, messageTitle.height)
+            Item
+            {
+                id: messageTypeIcon
+                visible: messageIcon.source != ""
+                height: UM.Theme.getSize("small_button_icon").height
+                width: visible ? UM.Theme.getSize("small_button_icon").height : 0
```

Comment:
This is not a button, nor should it align with any button. It seems appropriate to use `small_button_icon` here, then?

Reply:
Changed to it's own theme entry.